_posts/0000-01-02-Shreya-kt.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```

Comment:
#Timing purely depends on how clear your decisions are 
##So there is no point blaming the fate

> #Timing purely depends on how clear your decisions are 
##So there is no point blaming the fate